frame/support/procedural/src/benchmark.rs

```diff
@@ -886,11 +875,6 @@ fn expand_benchmark(
 		// benchmark function definition
 		#fn_def
 
-		// compile-time assertions that each referenced param type implements ParamRange
-		#(
-			#home::assert_impl_all!(#param_types: #home::ParamRange);
```

Comment:
so let's replace this with a compile-time assertion that `start` and `end` are both bounded by `Into<u32>`